packages/wallet/src/redux/sagas/message-sender.ts

```diff
@@ -34,6 +34,8 @@ function* createResponseMessage(action: JsonRpcResponseAction) {
         turnNum,
         channelId
       });
+    case "WALLET.UPDATE_CHANNEL_RESPONSE":
```

Comment:
Could we add test cases for these to `message-sender.test.ts`?

Reply:
I think I did add [one here](https://github.com/statechannels/monorepo/pull/477/files#diff-7ee60bb538c4c14602a93712ecb92032R296)

Reply:
I think that test checks that the `message-handler` calls the `message-sender` with the correct `message`.

The `message-sender.test.ts` makes sure that the `message-sender` knows how to handle the message. (IE: If `message-sender` gets called with `WALLET.UPDATE_CHANNEL_RESPONSE` does it attempt to `postMessage` a proper json-rpc response. ) Mostly a sanity check.

Reply:
Oh sorry I misread the name of the file. Adding